frontend/src/rootReducer.js

```diff
@@ -13,6 +13,7 @@ import * as plural from 'core/plural';
 import * as project from 'core/project';
 import * as resource from 'core/resource';
 import * as stats from 'core/stats';
+import * as terms from 'core/term';
```

Comment:
It's better to match the actual module name, otherwise it might be confusing for future readers. Especially since there is actually a `terms` module.